python/pip_install/parse_requirements_to_bzl/__init__.py

```diff
@@ -7,23 +7,40 @@
 from python.pip_install.extract_wheels.lib import bazel, arguments
 from pip._internal.req import parse_requirements, constructors
 from pip._internal.req.req_install import InstallRequirement
+from pip._internal.req.req_file import get_file_content, preprocess, handle_line, get_line_parser, RequirementsFileParser
 from pip._internal.network.session import PipSession
 
 
-def parse_install_requirements(requirements_lock: str) -> List[InstallRequirement]:
-    return [
-        constructors.install_req_from_parsed_requirement(pr)
-        for pr in parse_requirements(requirements_lock, session=PipSession())
-    ]
+def parse_install_requirements(requirements_lock: str, extra_pip_args: List[str]) -> List[Tuple[InstallRequirement, str]]:
+    ps = PipSession()
+    # This is roughly taken from pip._internal.req.req_file.parse_requirements in order to keep
```

Comment:
thanks for the reference!
nit: even better to link to the sources you're talking about at the specific version, I wouldn't know where to find this

Reply:
Will do.